mitosheet/src/jupyter/api.tsx

```diff
@@ -67,8 +66,29 @@ export enum UserJsonFields {
     UJ_RECEIVED_CHECKLISTS = 'received_checklists',
 }
 
+interface MitoSuccessOrInplaceErrorResponse {
+    'event': 'response',
```

Comment:
I guess `response` is about as precise as we can get. Maybe `successful_response`?

Reply:
It ain't. It's ether a Success or Inplace Error response, per the name of the variable...